source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -40,45 +42,75 @@ guide, consider importing the following classes in your program:
    import static com.mongodb.client.model.mql.MqlValues.*;
    import static java.util.Arrays.asList;
 
-The examples use the ``Arrays.asList()`` method to create a list of
-aggregation operations that can be passed as a pipeline to the
-``aggregate()`` method.
+To access document fields in an expression, you need to reference the
+document being processed by the aggregation pipeline. Use the
+``current()`` method to refer to this document. To access the value of a
+field, you must use the appropriately typed method, such as
+``getString()`` or ``getDate()``. When you specify the type for a field,
+you ensure that the driver recognizes its type correctly and lets you
+use methods that are compatible with that type. The following code
```

Comment:
```suggestion
you ensure that the driver provides only those
methods that are compatible with that type. The following code
```
(Avoiding the "recognition" (checking) and "letting" concepts since they are mildly misleading - for example, driver can't check types.)